Entity/Concrete/ProductDetails.cs

```diff
@@ -8,7 +9,8 @@ public class ProductDetails : BaseEntity
 
     public decimal Price { get; set; }
 
-    public string ImageUrl { get; set; }
+    public IFormFile? ImageFile { get; set; }
+    public string ImagePath { get; set; }
```

Comment:
Don't forget to add empty row between properties as a separator, please

Reply:
Can you [this](https://github.com/osmanaliyardim/WebNetSample/pull/7#:~:text=%E2%80%A6-,dc9b231,-Merge%20state) commit?